projects/github-actions/repo-gardening/src/utils/openai/send-request.js

```diff
@@ -0,0 +1,42 @@
+const { getInput } = require( '@actions/core' );
+const OpenAI = require( 'openai' );
+const debug = require( '../debug' );
+
+/**
+ * Send a message to OpenAI.
+ *
+ * @param {string} message - Message to send to OpenAI.
+ *
+ * @return {Promise<string>} Promise resolving to the response from OpenAI.
+ */
+async function sendOpenAiRequest( message ) {
+	const apiKey = getInput( 'openai_api_key' );
+	if ( ! apiKey ) {
+		debug( 'openai: Input openai_api_key is required but missing. Aborting.' );
+		return '';
```

Comment:
Totally optional, but this seems like something that should throw instead of just returning an empty string.

Reply:
That's how I had set it up at first, but I figured it may not be ideal for folks outside of a8c who will not be able to use this OpenAI service.

Reply:
I'd add a check for whether the key is set before calling this, or maybe a try-catch around the call.

Reply:
What would you think about something like this?
d92472e2931093685cb957325a7e4622f46402db

Reply:
Ok, left a few nitpicks inline.